x-pack/test_utils/enzyme_helpers.js

```diff
@@ -4,42 +4,103 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import PropTypes from 'prop-types';
+/**
+ * Components using the react-intl module require access to the intl context.
+ * This is not available when mounting single components in Enzyme.
+ * These helper functions aim to address that and wrap a valid,
+ * intl context around them.
+ */
+
 import React from 'react';
-import { mount } from 'enzyme'; // eslint-disable-line import/no-extraneous-dependencies
-import { intl } from './mocks/intl';
+import { I18nProvider, intlShape } from '@kbn/i18n/react';
+import { mount, shallow, render } from 'enzyme'; // eslint-disable-line import/no-extraneous-dependencies
+
+let intl;
 
 /**
- *  Creates the wrapper instance with provided intl object into context
- *
- *  @param  node The React element or cheerio wrapper
- *  @param  options properties to pass into mount wrapper
- *  @return The wrapper instance around the rendered output with intl object in context
+ * This component is used to get intl object passed by IntlProvider via context
  */
-export function mountWithIntl(node, { context = {}, childContextTypes = {}, ...props } = {}) {
-  if (!node) {
-    throw new Error(`First argument should be cheerio object or React element, not ${node}`);
+class IntlGetter extends React.Component {
```

Comment:
__suggestion:__ I played with this code a bit, and it seems that something like the code below will work just fine and will be shorter:
```js
// Use fake component to extract `intl` property to use in tests.
const { intl } = mount(<I18nProvider><br /></I18nProvider>)
  .find('IntlProvider').instance().getChildContext();
```

Reply:
Used your solution. thanks!